sample/kotlin/build.gradle.kts

```diff
@@ -160,7 +161,7 @@ dependencies {
     // Local Storage
     implementation(libs.sqlDelight)
     implementation(libs.room)
-    kapt(libs.roomCompiler)
+    ksp(libs.roomCompiler)
```

Comment:
the only remaining usage of `kapt` is by Glide, but it doesn't support `ksp` yet.